backend/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
```

Comment:
Can you elaborate on what this file is for? How does it connect to the work Janet is doing on the backend?

Reply:
I am not sure what this file is for yet, I believe it occurred within this PR due to me formatting it

Reply:
Hmm .. candidate for deletion then?

Reply:
Sure